src/testHelpers/jest-setup.js

```diff
@@ -18,15 +18,9 @@ global.fetch = fetch;
 global.shallow = shallow;
 global.mount = mount;
 
-process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN =
-  'http://localhost.bbc.com:7080';
+process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN = 'http://localhost:7080';
 process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = '/';
 process.env.SIMORGH_ASSETS_MANIFEST_PATH = path.resolve(
   __dirname,
   '../server/assets/fixture.json',
 );
-
-process.env.SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN =
-  'http://localhost.bbc.com:7080';
-
-process.env.SIMORGH_PUBLIC_STATIC_ASSETS_PATH = '/';
```

Comment:
These duplicate lines appear to have been added in https://github.com/bbc/simorgh/pull/4175 reverting https://github.com/bbc/simorgh/pull/4117